presto-docs/src/main/sphinx/release/release-0.268.rst

```diff
@@ -0,0 +1,31 @@
+=============
+Release 0.268
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add support to use consistent hashing as a hash strategy when affinity scheduling is used. This feature can be turned on by setting ``node-scheduler.node-selection-hash-strategy`` to ``CONSISTENT``. A separate configuration ``node-scheduler.consistent-hashing-min-virtual-node-count`` specifies the minimal number of virtual nodes in the consistent hashing ring.
```

Comment:
Ideally, we would link to documentation for something like this.  "Consistent hashing" may not be immediately obvious to a Presto administrator.

Reply:
How do you recommend adding documentation link here? Any suggestion?